src/solver/application/include/antares/application/application.h

```diff
@@ -110,6 +113,10 @@ class Application final : public Yuni::IEventObserver<Application, Yuni::Policy:
                        Benchmarking::IDurationCollector& duration_collector);
 
     void writeComment(Data::Study& study);
+    void startSimulation(Data::StudyLoadOptions options);
```

Comment:
by value? is it moved later?

Reply:
Indeed, `startSimulation` makes a copy while `readDataForTheStudy` works on a non-const ref

```cpp
void Application::readDataForTheStudy(Data::StudyLoadOptions& options);

void Application::startSimulation(Data::StudyLoadOptions options)
{
// ...
    readDataForTheStudy(options); // Works on a copy of
// ..
}
```
It should work but I find it a bit strange.